substrate/runtime/src/lib.rs

```diff
@@ -158,10 +173,10 @@ impl pallet_balances::Config for Runtime {
   type DustRemoval = ();
   type ExistentialDeposit = ConstU64<500>;
   type AccountStore = System;
-  type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
```

Comment:
why the move away from pallet_ convention?

Reply:
The line you replied to doesn't.

In general, I've used `_pallet` for three reasons.

1) `pallet` is under X. X is not under `pallet`. X has a pallet.
2) It matches our folder layout, which uses the above commentary to define as `X/{primitives, pallet, client}`, unlike Substrate, minimizing travel distance/hunting length between the parts of a given project (no need to open a series of other folders/scroll all the way through 100 other pallets just to get to the pallet for the client you're working on).
3) It immediately says it's a Serai pallet, not a Substrate pallet. Tendermint bucks this convention as Tendermint was meant to be usable as a Substrate pallet, per https://github.com/serai-dex/serai/issues/137.